src/NzbDrone.Core/Notifications/CustomScript/CustomScript.cs

```diff
@@ -20,11 +21,15 @@ public class CustomScript : NotificationBase<CustomScriptSettings>
         private readonly IProcessProvider _processProvider;
         private readonly Logger _logger;
 
-        public CustomScript(IDiskProvider diskProvider, IProcessProvider processProvider, Logger logger)
+        // Have to use _tagRepo instead of _tagService because the service creates a crashing loop :|
+        private readonly ITagRepository _tagRepo;
```

Comment:
Separate TagService into TagService and TagDetailsService. Then you should be able to call TagService from here.